processor/transform/reusable-content.js

```diff
@@ -5,23 +5,20 @@ export const type = 'reusable-content';
 const regexp = /^\s*<(?<tag>[A-Z]\S+)\s*\/>\s*$/;
 
 const reusableContentTransformer = function () {
-  const { tags, disabled } = this.data('reusableContent');
+  const { tags, disabled, writeTags = true } = this.data('reusableContent');
   if (disabled) return () => undefined;
 
   return tree => {
     visit(tree, 'html', (node, index, parent) => {
       const result = regexp.exec(node.value);
       if (!result || !result.groups.tag) return;
-
       const { tag } = result.groups;
 
-      const block = {
-        type,
-        tag,
-        children: tag in tags ? tags[tag] : [],
-      };
-
-      parent.children[index] = block;
+      if (writeTags) {
+        parent.children[index] = { type, tag, children: tag in tags ? tags[tag] : [] };
+      } else {
+        parent.children[index] = tag in tags ? [...tags[tag]].pop() : {};
```

Comment:
What's the `pop` for?

Reply:
`tags[tag]` comes back as a node in an array, and i like `.pop()` better than `[0]` 😂

Reply:
I think that may be dangerous though, the RC block could have multiple items in the array?

Reply:
I think you should probably splice in the whole array.

Reply:
ahh you're right! good catch!